# multistep walk params
bool use_demo_walk   # if true, will walk in figure 8
atlas_msgs/AtlasBehaviorStepParams[] multistep_params
